<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>research | Yiqun T. Chen</title> <meta name="author" content="Yiqun T. Chen"> <meta name="description" content="Data Science Postdoctoral Fellow, Stanford University "> <meta name="keywords" content="Data Science, Biostatistics, LLM for Science"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://yiqunchen.github.io/research/"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?16c85d7ef3cb8b6d0fce08727934ec0a"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Yiqun </span>T. Chen</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item active"> <a class="nav-link" href="/research/">research<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">research</h1> <p class="post-description"></p> </header> <article> <p>My work centers on <em>bringing statistical insight and understanding to the practice of data science</em>. As a classically-trained statistician with an extensive track record of collaborative work, industry experience, as well as experience with modern machine learning methods, I seek to identify the challenges that arise from contemporary data-driven inquiry, and to develop statistical frameworks to solve them.</p> <h3 id="data-science-for-generative-ai">Data science for generative AI:</h3> <p>During my postdoctoral training, I am particularly enthusiastic about exploring the data science inquires arising from advanced AI techniques, such as large language models (LLMs) and text-to-image models (e.g., DALLE and Stable Diffusion).</p> <p>One line of my work applies large language models (LLMs) to single-cell genomics: While there has been significant recent progress in foundation models for single-cell data, existing models require extensive data curation and resource-intensive training on tens of millions of cells. We explored a much simpler alternative, GenePT, by leveraging LLMs embeddings of genes and cells based on descriptions of their functionalities in the literature. On many downstream tasks, GenePT achieves comparable, and often better, performance than existing single-cell foundation models.</p> <p>I have also introduced TWIGMA, a comprehensive dataset of generative-AI images on Twitter. We found that gen-AI images possess distinctive characteristics and exhibit, on average, lower variability when compared to their non-gen-AI counterparts. Additionally, we characterize the longitudinal shift in the themes of AI-generated images by users.</p> <h4 id="selected-manuscripts">Selected Manuscripts:</h4> <ul> <li> <strong>Chen YT</strong>, Zou J (2023+). A Simple But Hard-to-Beat Foundation Model for Genes and Cells Built From ChatGPT. <ul> <li>Preprint: <a href="https://www.biorxiv.org/content/10.1101/2023.10.16.562533v1" rel="external nofollow noopener" target="_blank">https://www.biorxiv.org/content/10.1101/2023.10.16.562533v1</a> </li> <li>A preliminary of this work has been accepted as an oral presentation at MLCB 2023 (&lt; 15% acceptance rate).</li> </ul> </li> <li> <strong>Chen YT</strong>, Zou J (2023). TWIGMA: A dataset of AI-Generated Images with Metadata From Twitter. NeurIPS 2023. arXiv link: <a href="https://arxiv.org/abs/2306.08310." rel="external nofollow noopener" target="_blank">https: //arxiv.org/abs/2306.08310</a>.</li> </ul> <h3 id="testing-data-driven-hypotheses">Testing data-driven hypotheses</h3> <p>In my PhD research, I tackled the problem of testing <em>data-driven hypotheses generated by unsupervised learning methods</em> such as changepoint detection and clustering. For instance, in single-cell RNA-sequencing analyses, researchers often first cluster the cells, and then test for a difference in the expected gene expression levels between the clusters. However, this is invalid from a statistical perspective: once we have used the data to generate hypotheses, standard statistical inference tools will no longer be valid (e.g., <em>an extremely inflated Type I error rate</em>). I have developed valid inference approaches in this setting, and have applied them to data from genomics, neuroscience, and epidemiology.</p> <h4 id="selected-manuscripts-1">Selected Manuscripts:</h4> <ul> <li> <p>Clustering And Differential Expression Testing: <strong>Chen YT</strong>, Gao LL (2023+). Testing for a difference in means of a single feature after clustering. arXiv link.</p> </li> <li> <p>Inference after <strong>k-means clustering</strong>: <strong>Chen YT</strong>, Witten DM (2023). Selective inference for k-means clustering. <em>Journal of Machine learning Research</em>. arXiv link: <a href="https://arxiv.org/abs/2203.15267" rel="external nofollow noopener" target="_blank">https://arxiv.org/abs/2203.15267</a>.</p> </li> <li> <p>Inference after <strong>graph fused lasso</strong>: <strong>Chen YT</strong>, Jewell SW, and Witten DM (2023). <em>Journal of Computational and Graphical Statistics</em>. arXiv link: https://arxiv.org/abs/2109.10451.</p> </li> <li> <p>Inference after <strong>spike detection</strong>: <strong>Chen YT</strong>, Jewell SW, Witten DM. (2022) <a href="https://doi.org/10.1093/biostatistics/kxab034" rel="external nofollow noopener" target="_blank">Quantifying uncertainty in spikes estimated from calcium imaging data</a>, <em>Biostatistics</em>, Oxford University Press.</p> </li> </ul> <h4 id="selected-software">Selected Software:</h4> <ul> <li>CADET: <a href="https://yiqunchen.github.io/CADET/">https://yiqunchen.github.io/CADET/</a> </li> <li>KmeansInference: <a href="https://yiqunchen.github.io/KmeansInference/">https://yiqunchen.github.io/KmeansInference/</a> </li> <li>GFLassoInference: <a href="https://yiqunchen.github.io/GFLassoInference/">https://yiqunchen.github.io/GFLassoInference/</a> </li> <li>SpikeInference: <a href="https://github.com/yiqunchen/SpikeInference" rel="external nofollow noopener" target="_blank">https://github.com/yiqunchen/SpikeInference</a> </li> </ul> <h4 id="selected-talks">Selected Talks:</h4> <ul> <li> <p>You can learn more about my work on “selective inference for k-means clustering” at International Seminar on Selective Inference <a href="https://drive.google.com/file/d/1_oHTmKxO3exiOFhNfE56VJk7OXvGkoCd/view?usp=sharing" rel="external nofollow noopener" target="_blank">here</a></p> </li> <li> <p>You can learn more about my work on “Quantifying uncertainty in spikes estimated from calcium imaging data” at WNAR 2021 <a href="https://youtu.be/tMzv-ok4Liw" rel="external nofollow noopener" target="_blank">here</a></p> </li> </ul> <p><img src="https://yiqunchen.github.io/images/Figure_6_b.png" style="display: block; margin: auto; max-width: 75%;"></p> <h3 id="selected-applications-to-biomedical-data-science">Selected applications to (biomedical) data science:</h3> <h4 id="race-and-ethnicity-in-research-data-collection">Race and Ethnicity in Research Data Collection:</h4> <p>In today’s data-driven era, statisticians should play an active role in study design and data collection. Collaborating with researchers from human-computer interaction (HCI) - a field crucial for spawning innovative technologies and interfaces — and computing ethics, I investigated the racial and ethnic diversity of research participants in Human-Computer Interaction (HCI). Our work spotlighted a conspicuous absence of the documentation of participants’ racial and ethnic backgrounds. Together with colleagues, I consequently formulated guidelines for HCI researchers regarding the collection of race and ethnicity data, and emphasized the pressing need for conscious decision-making rather than passive omissions, ensuring the creation of truly inclusive and equitable technologies.</p> <p><img src="https://yiqunchen.github.io/images/chi_research_statement_fig.png" style="display: block; margin: auto; max-width: 40%; float:right"></p> <p>Selected Publications:</p> <ul> <li> <strong>Chen YT</strong>, Smith AD, Reinecke K, and To A (2023). Why, when, and from whom: considerations for collecting and reporting race and ethnicity data in HCI. In CHI’23. <ul> <li>Awarded Best Paper Honorable Mention (Top 5% of all submissions to CHI 2023); also covered in https://www.khoury.northeastern.edu/awards-ethics-cross-college-collaboration-northeastern-at-chi-2023.</li> <li>This paper reviewed six years of CHI proceedings and interviewed 15 authors to understand the current practice of collecting race and ethnicity data in the field of Human-computer Interaction. It highlights important considerations for researchers when they decide to collect such data.</li> </ul> </li> <li> <strong>Chen YT</strong>, Smith AD, Reinecke K, and To A (2022). Collecting and Reporting Race and Ethnicity Data in HCI. In CHI’22 Extended Abstracts.</li> </ul> <h4 id="social-networks-characteristics-and-hivtb-care-outcomes-in-sub-saharan-africa">Social networks characteristics and HIV/TB care outcomes in sub-Saharan Africa</h4> <p>Social isolation among HIV-positive persons might be an important barrier to care. Similarly, analyzing one’s social network could elucidate Tuberculosis (TB) transmission dynamics outside of their home and may inform novel, network-based case-finding strategies.</p> <p>We used data collected as a part of the <a href="https://www.searchendaids.com/" rel="external nofollow noopener" target="_blank">SEARCH Study</a> in rural Kenya and Uganda, and constructed 32 community-wide, sociocentric networks. We found that (i)HIV-positive persons named as a contact by fewer people may be at higher risk for poor HIV care outcomes,suggesting opportunities for targeted interventions; and (ii) social networks with higher centrality, more men, contacts with HIV, and TB infection, were positively associated with TB infection.</p> <p><img src="https://yiqunchen.github.io/images/cropped_network.png" style="display: block; margin: auto; max-width: 35%; float:right"></p> <p>Selected Publications:</p> <ul> <li> <p>TB infections and social networks: <strong>Chen YT</strong><sup>†</sup>, Marquez C<sup>†</sup>, Atukunda M, Chamie G, Balzer LB, …, Charlebois ED, Havlir DV, Petersen ML (2022+). The Association Between Social Network Characteristics and TB Infection Among Adults in Nine Rural Ugandan Communities. To appear in <em>Clinical Infectious Diseases</em>; † denotes joint first authorship.</p> </li> <li> <p>HIV care outcome and social networks: <strong>Chen YT</strong>, Brown LB, Chamie G, Kwarisiima D, Ayieko J, Kabami J, Charlebois E, Clark T, Kamya M, Havlir DV, Petersen ML, and Balzer LB (2021). Social networks and HIV care outcomes in rural Kenya and Uganda. <em>Epidemiology</em>, 32(4):551-559.</p> </li> <li> <p>Two posters at CROI 2020:</p> <ul> <li>Project 1: <a href="https://www.croiconference.org/abstract/hiv-persons-in-rural-uganda-with-fewer-social-connections-have-lower-hiv-suppression/" rel="external nofollow noopener" target="_blank">Preliminary work of this manuscript</a> </li> <li>Project 2: <a href="https://www.croiconference.org/abstract/using-social-networks-to-reach-individuals-with-low-cd4-at-high-risk-of-death/" rel="external nofollow noopener" target="_blank">Using social networks to reach individuals with low CD4 counts</a> </li> </ul> </li> </ul> <h4 id="software-engineering">Software engineering:</h4> <p>The relationship between fault detection, test adequacy criteria, and test set size remains a complex yet central topic in software engineering research. In this project, we addressed the supposed <em>contradiction</em> of prior work and explains why test set size is neither a confounding variable, as previously suggested, nor an independent variable that should be experimentally manipulated. An alternative methodology is proposed for comparing test adequacy criteria on an equal basis, which accounts for test set size without directly manipulating it through unrealistic stratification.</p> <p>Manuscript:</p> <ul> <li> <strong>Chen YT</strong>, Gopinath R, Tadakamalla A, Ernst MD, Holmes R, Fraser G, Am- mann P, Just R. Revisiting the relationship between fault detection, test adequacy criteria, and test set size. In: 2020 35th IEEE/ACM International Conference on Automated Software Engineering (ASE). 2020:237-249.</li> <li>Link to the publisher’s version <a href="https://ieeexplore.ieee.org/document/9286102" rel="external nofollow noopener" target="_blank">here</a>.</li> <li>You can learn more about this project by watching my talk at ASE 2020 <a href="https://homes.cs.washington.edu/~rjust/publ/mutants_faults_revisited_ase_2020.mp4" rel="external nofollow noopener" target="_blank">here</a> <img src="https://yiqunchen.github.io/images/test_slide.png" style="display: block; margin: auto; max-width: 70%;"> </li> </ul> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Yiqun T. Chen. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?07b8786bab9b4abe90d10e61f7d12ff7" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>